concordia/language_model/utils.py

```diff
@@ -32,6 +32,7 @@ def language_model_setup(
     model_name: str,
     api_key: str | None = None,
     disable_language_model: bool = False,
+    device: str = 'cpu'
 ) -> language_model.LanguageModel:
   """Get the wrapped language model.
 
```

Comment:
Please add the doc for the device argument.

Reply:
Done, thanks for flagging